client/post-editor/controller.js

```diff
@@ -115,6 +115,7 @@ module.exports = {
 
 		function startEditing( siteId ) {
 			context.store.dispatch( setEditorPostId( postID ) );
```

Comment:
Yes, so I added the `startPostEditor` action in 4c6278d but I kept the cleaning on unmount, because I was experiencing the same bug in #4539. SO any thoughts about the current implementation? Is this new Action worth adding? It looks ok, but the initial change was 4 lines of code :P